README.md

```diff
@@ -760,6 +762,9 @@ Notes:
  * Notes on Multipack
     - works only for *multi-gpu*.
     - currently only includes the version of *multipack* optimized for linear attention implementations like *flash-attn*.
+ * Notes of Fast MOE
+    - `--fast_moe` is an integer value that configures the amount of expert parallel sharding (ep_degree).
+    - Running fast moe modifies the state dict of the model, and must be post-processed using [checkpoint utils](https://github.com/foundation-model-stack/fms-acceleration/blob/main/plugins/accelerated-moe/src/fms_acceleration_moe/utils/checkpoint_utils.py) to run inference.
```

Comment:
To run HF or vLLM inference it requires this post-processing? Can you explain more on the usage of the script, perhaps providing an example command to run here